charts/nexus-iq/values.yaml

```diff
@@ -90,19 +90,17 @@ service:
 
 ingress:
   enabled: false
-  annotations: {kubernetes.io/ingress.class: nginx}
-    # kubernetes.io/ingress.class: nginx
-    # kubernetes.io/tls-acme: "true"
+  annotations: {}
   hostUI: iq-server.demo
   hostUIPath: /
   hostAdmin: admin.iq-server.demo
   hostAdminPath: /
 
-  tls: []
-    # - secretName: nexus-local-tls
-    #   hosts:
-    #     - iqserver.local
-    #     - admin.iqserver.local
+  # tls:
+  #   - secretName: nexus-local-tls
+  #     hosts:
+  #       - iq-server.demo
+  #       - admin.iq-server.demo
```

Comment:
these hosts should have matched the host names used for the endpoints, so now they do.